### ISSUE TITLE
Create/Transfer Ocean Token

### ISSUE THREAD

**Issue by OceanMan23** [oceanprotocol/contracts#758]
Hello all,

I have a question.
The Contract have been passed to the Ganache (local setup with barge), we have also received an Ocean Contract address in JSON. The Ocean address was added in the metamask as a token address. The account balance is 100 for ETH and 0 for the Ocean token.

How can we transfer a Ganach account some Ocean tokens? Unfortunately, we are not getting anywhere.

Many greetings and best thanks

![image](https://github.com/oceanprotocol/contracts/assets/135710575/8668033c-b99a-4d13-b07e-be2b1478b6e5)

**Comment by trentmc** [comment]
Hello,

Please follow the local setup steps at this README: https://github.com/oceanprotocol/ocean.py/blob/main/READMEs/setup-local.md

This includes console lines to specify accounts TEST_PRIVATE_KEY1 and TEST_PRIVATE_KEY2.

And it includes Python code to mint OCEAN to those accounts: 
```python
from ocean_lib.ocean.mint_fake_ocean import mint_fake_OCEAN
mint_fake_OCEAN(config)
```

**Comment by OceanMan23** [comment]
Hello trentmc, 

thank you very much for the feedback. I followed the instructions. I took the TEST_PRIVATE_KEY1 and TEST_PRIVATE_KEY2 from Ganach. 

Which account key should I set for FACTORY_DEPLOYER_PRIVATE_KEY? 
Note: The Smart-Contract container starts up with MNEMONIC and creates the contracts on Ganach. 

I get the following error message on the console: "brownie.exceptions.VirtualMachineError: revert: Ownable: caller is not the owner".

Thanks a lot
